src/bp/ui-studio/src/web/components/Layout/StatusBar/BottomPanel.tsx

```diff
@@ -128,15 +130,21 @@ class BottomPanel extends React.Component<IProps, IState> {
             </Tooltip>
 
             <Tooltip content={<em>Download logs</em>}>
-              <Button minimal={true} icon={'import'} small={true} type="button" onClick={this.handleDownloadLogs} />
+              <Button icon={'import'} small={true} type="button" onClick={this.handleDownloadLogs} />
+            </Tooltip>
+
+            <Divider />
+
+            <Tooltip content={<em>Clear log history</em>}>
+              <Button icon={'trash'} small={true} type="button" onClick={() => this.props.setLogs([])} />
```

Comment:
I think logs should be cleared after X number accumulated as well as leaving this clear log button.

Reply:
Yep, it keeps only the last 200 logs, I think it could be configurable in the future. When there is may of them, the browser is a bit slow